Dockerfile

```diff
@@ -1,10 +1,13 @@
-FROM python:3.12-slim
-EXPOSE 8000
-
+FROM python:3.12-slim as build
 COPY . /occameracontrol
-WORKDIR /occameracontrol
 
-RUN pip install --no-cache-dir -r /occameracontrol/requirements.txt
+FROM python:3.12-slim
+EXPOSE 8000
+RUN --mount=type=bind,from=build,source=/occameracontrol,target=/occameracontrol \
```

Comment:
I am not really sure about the multiple build stages: is this just a fancy way of avoiding to delete the temporary files or does this do something else?

Reply:
It may be a bit overboard here and may not matter much, but it actually does a bit more.
If you do something like 
```Dockerfile
COPY . /occameracontrol
RUN pip install --no-cache-dir .
RUN rm -rf /occameracontrol
```

…you end up with multiple layers of the container and while the last layer doesn't contain the files any longer, the others still do. You can somewhat circumvent this by using one `RUN` command which contains a chain of command, cleaning up in the same step, but that doesn't work if you want to use `COPY` first like I did here.

Thus, using stages is a fancy way of actually removing the source files.

That being said, it is probably a bit overkill since the sources are quite small. Nevertheless, this should create somewhat cleaner containers.